src/utils/parse-arguments-into-options.ts

```diff
@@ -101,6 +101,11 @@ export async function parseArgumentsIntoOptions(
     );
   }
 
+  if (project && /\s$/.test(project)) {
+    project = null;
+    console.log(chalk.yellow(" Project name cannot end with whitespace. Enter a valid project name."));
```

Comment:
I think we can remove this? Since this is not shown to the user right?  

Also maybe just add a comment about regex saying it checks for any tralling spaces in end? 

or maybe move that regex to some declarative var so that we can use it in   `prompt-for-missing-options` too?

Reply:
> I think we can remove this? Since this is not shown to the user right?

It's shown to the user, see yellow text in the video (from 0:02) and also screenshot

> or maybe move that regex to some declarative var so that we can use it in prompt-for-missing-options too?

Thanks!  [f99426a](https://github.com/scaffold-eth/create-eth/pull/176/commits/f99426a0c50e88cecdeccd465e5d2c312c32687e)

Reply:
Ohhh yess sorry my bad! So it's shown when you pass the directory name directly while initiating the cli